pkg/apis/app/v2/kieapp_types.go

```diff
@@ -240,6 +240,8 @@ type KieAppObject struct {
 	ImageTag string `json:"imageTag,omitempty"`
 	// The storageClassName to use
 	StorageClassName string `json:"storageClassName,omitempty"`
+	// The Size of the PV
+	PvSize string `json:"pvSize,omitempty"`
```

Comment:
If this is something that should only be configurable for the Console deployment, we may want to move it to the `ConsoleObject` struct instead. This `KieAppObject` is for common fields used by all object types... servers, smartrouters, etc. as well.

If instead you believe this should stay where it is, we should add support for the other object types as well in our templates. Otherwise, it will confuse the user to see it configurable in the server / router schema if it doesn't do anything.

Reply:
updated  it.